example.settings.yaml

```diff
@@ -1,29 +1,32 @@
 telegram:
-  token: TELEGRAM_BOT_TOKEN_HERE      # Telegram bot tokens look like this: 113031530:AAH_4c5Gt9gsVu82OVNjUV7poU7PodYiNsA
+  token: TELEGRAM_BOT_TOKEN_HERE # Telegram bot tokens look like this: 113031530:AAH_4c5Gt9gsVu82OVNjUV7poU7PodYiNsA
   useFirstNameInsteadOfUsername: false
   colonAfterSenderName: false
   skipOldMessages: true
   sendEmojiWithStickers: true
 discord:
   useNickname: false
-  token: DISCORD_BOT_TOKEN_HERE       # Discord bot tokens look like this: MjI3MDA1NzIvOBQ2MzAzMiMz.DRf-aw.N0MVYtDxXYPSQew4g2TPqvQve2c
+  token: DISCORD_BOT_TOKEN_HERE # Discord bot tokens look like this: MjI3MDA1NzIvOBQ2MzAzMiMz.DRf-aw.N0MVYtDxXYPSQew4g2TPqvQve2c
   skipOldMessages: true
   replyLength: 100
   maxReplyLines: 2
 bridges:
   - name: First bridge
     direction: both
     telegram:
-      chatId: TELEGRAM_CHAT_ID  # Remember that Telegram group and channel IDs are negative. Include the `-` if you are bridging a group or a channel
+      chatId: TELEGRAM_CHAT_ID # Remember that Telegram group and channel IDs are negative. Include the `-` if you are bridging a group or a channel
       relayJoinMessages: true
       relayLeaveMessages: true
       sendUsernames: true
       relayCommands: true
       crossDeleteOnDiscord: true
     discord:
-      channelId: 'DISCORD_CHANNEL_ID'       # This ID must be wrapped in single quotes. Example: '244791815503347712'
+      channelId: "DISCORD_CHANNEL_ID" # This ID must be wrapped in single quotes. Example: '244791815503347712'

```

Comment:
Need to either change the quotes or the comment here, otherwise somewhat confusing

Reply:
I don't recall changing this, I'll put it back and change the other string for consistency